### PR TITLE
Compatibility with Rails 4.1 and 4.2

### PR DESCRIPTION
**Pull request by pschambacher**
cc @grosser @staugaard

### CODE REVIEW

**Comment by pschambacher** [comment]
I had to drop Rails 3.2 on Ruby 2.2 since it's not supported

File: .travis.yml [hunk 1]
```diff
@@ -2,7 +2,14 @@ rvm:
...
+  - 2.2
 gemfile:
-  - gemfiles/rails_3.2.gemfile
-  - gemfiles/rails_4.0.gemfile
+  - gemfiles/rails3.2.gemfile
+  - gemfiles/rails4.0.gemfile
+  - gemfiles/rails4.1.gemfile
+  - gemfiles/rails4.2.gemfile
+matrix:
+  exclude:
```
Comment (grosser) [inline .travis.yml]: damn here we go again with matrix excludes ...

File: .travis.yml [hunk 2]
```diff
@@ -2,7 +2,14 @@ rvm:
   - 1.9
```
Comment (grosser) [inline .travis.yml]: 1.9 / 2.0can go afaik

**Comment by grosser** [comment]
:+1:

**Comment by staugaard** [comment]
:+1:

**Comment by pschambacher** [comment]
I bumped but I don't have the rights to release a new version, probably you @staugaard :)

**Comment by staugaard** [comment]
done

On Thu, Feb 5, 2015 at 1:11 AM, Pierre Schambacher <notifications@github.com
> wrote:

> I bumped but I don't have the rights to release a new version, probably
> you @staugaard <https://github.com/staugaard> :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/stronger_parameters/pull/12#issuecomment-73015335>
> .
>



-- 
Thanks,
Mick Staugaard
Senior Principal Engineer

Zendesk <http://www.zendesk.com/>
Github: @staugaard

**Pull request merged**